### PR TITLE
[editorial] fix wrong or missing use of code brackets

### PR DESCRIPTION
**Pull request by egekorkan**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/wot-thing-description/pull/1230.html" title="Last updated on Sep 21, 2021, 10:50 AM UTC (b77006a)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-thing-description/1230/37a0a29...b77006a.html" title="Last updated on Sep 21, 2021, 10:50 AM UTC (b77006a)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4089,7 +4089,7 @@ <h3><code>forms</code></h3>
 
   <p>
   When <code>forms</code> is present at the top level, it can be used to describe meta interactions offered by a <a>Thing</a>.
-  For example, the operation types "readallproperties" and "writeallproperties" are for meta
+  For example, the operation types <code>"readallproperties"</code> and <code>writeallproperties"</code> are for meta
```
Comment (danielpeintner) [inline index.html:4092]: Q1: Do we want surrounding quotes " ?
Q2: IF we want quotes the second one misses it. see <code>writeallproperties"

Comment (egekorkan) [reply index.html:4092]: Ah it should be without the quotes, fixing

**Review by danielpeintner — APPROVED** [review]
LGTM

**Comment by sebastiankb** [comment]
from today's TD call group decided to merge

**Pull request merged**